<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Edit_User_Data</class>
 <widget class="QWidget" name="Edit_User_Data">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>320</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Edit User</string>
  </property>
  <property name="windowIcon">
   <iconset resource="res.qrc">
    <normaloff>:/icons/icons/library.png</normaloff>:/icons/icons/library.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget
{
	background-color: rgb(244, 241, 222);
	color: rgb(61, 64, 91);
}
QPushButton
{
	font: 10pt &quot;IRANYekan&quot;;
	background-color: rgb(129, 178, 154);
	color: rgb(255, 255, 255);
    border: 2px solid rgb(255, 255, 255);
    border-radius: 6px;
}
QPushButton:hover
{
	background-color: rgb(242, 204, 143);
}</string>
  </property>
  <widget class="QFrame" name="frame_top">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>480</width>
     <height>100</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(61, 64, 91);
color: rgb(255, 255, 255);</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QFrame" name="frame_icon">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">image: url(:/icons/icons/conspectus.png);</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_title">
    <property name="geometry">
     <rect>
      <x>89</x>
      <y>25</y>
      <width>381</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 12pt &quot;IRANYekan&quot;;</string>
    </property>
    <property name="text">
     <string>Library Management System</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>89</x>
      <y>55</y>
      <width>311</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 25 8pt &quot;IRANYekan Light&quot;;</string>
    </property>
    <property name="text">
     <string>Edit Form (User) </string>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>41</x>
     <y>120</y>
     <width>391</width>
     <height>161</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="label_user">
      <property name="styleSheet">
       <string notr="true">font: 8pt &quot;IRANYekan&quot;;</string>
      </property>
      <property name="text">
       <string>Username:</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="lineEdit_user">
      <property name="styleSheet">
       <string notr="true">font: 8pt &quot;IRANYekan&quot;;</string>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_pass">
      <property name="styleSheet">
       <string notr="true">font: 8pt &quot;IRANYekan&quot;;</string>
      </property>
      <property name="text">
       <string>Password:</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="lineEdit_pass">
      <property name="styleSheet">
       <string notr="true">font: 8pt &quot;IRANYekan&quot;;</string>
      </property>
      <property name="echoMode">
       <enum>QLineEdit::Password</enum>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QRadioButton" name="radioButton_male">
      <property name="styleSheet">
       <string notr="true">font: 8pt &quot;IRANYekan&quot;;</string>
      </property>
      <property name="text">
       <string>Male</string>
      </property>
     </widget>
    </item>
    <item row="4" column="0" colspan="2">
     <widget class="QPushButton" name="pushButton_save">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>Save</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_sex">
      <property name="styleSheet">
       <string notr="true">font: 8pt &quot;IRANYekan&quot;;</string>
      </property>
      <property name="text">
       <string>Sex:</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QRadioButton" name="radioButton_female">
      <property name="styleSheet">
       <string notr="true">font: 8pt &quot;IRANYekan&quot;;</string>
      </property>
      <property name="text">
       <string>Female</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
